azure-pipelines.yml

```diff
@@ -104,10 +104,10 @@ stages:
         testResultsFiles: '**/tr.xml'
         testRunTitle: vstest
 
-  - job:
+  - job: t0_part1
     pool: sonictest
-    displayName: "kvmtest-t0"
-    timeoutInMinutes: 360
+    displayName: "kvmtest-t0-part1"
+    timeoutInMinutes: 240
```

Comment:
I think you can keep the original timeout value 360.

Reply:
Done